bedrock/firefox/tests/test_views.py

```diff
@@ -679,7 +679,7 @@ def test_better_browser_scene_1_non_us(self, render_mock):
         req = RequestFactory().get('/firefox/new/?xv=betterbrowser')
         req.locale = 'de'
         views.new(req)
-        render_mock.assert_called_once_with(req, 'firefox/new/scene1.html', ANY)
+        render_mock.assert_called_once_with(req, 'firefox/new/scene1.de.html', ANY)
```

Comment:
It turns out the reason these tests we're failing is becuase of the logic added in `views.py`. If you remove the lines in `views.py` and let the locale specific templates just do theri thing (see previous comment), then you shouldn't need to change these tests and they will still pass.